compiler/llvm/clangUtil.cpp

```diff
@@ -4955,6 +4969,11 @@ void makeBinaryLLVM(void) {
       }
     }
 
+    if (usingGpuLocaleModel() &&
+        getGpuCodegenType() == GpuCodegenType::GPU_CG_AMD_HIP) {
+      setenv("PATH", curPath.c_str(), /*override*/ 1);
+    }
+
```

Comment:
Why is this necessary? You do the same above when setting `curPath`?

Reply:
This part is setting `PATH` back to its original value. See the difference in arguments (`adjPath` vs `curPath`)